src/screens/NanoContractRegisterQrCodeScreen.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { View } from 'react-native';
+import { t } from 'ttag';
+
+import HathorHeader from '../components/HathorHeader';
+import QRCodeReader from '../components/QRCodeReader';
+import SimpleButton from '../components/SimpleButton';
+import { COLORS } from '../styles/themes';
+
+export const NanoContractRegisterQrCodeScreen = ({ navigation }) => {
+  const onSuccess = (e) => {
+    navigation.navigate('NanoContractRegisterScreen', { ncId: e.data });
+  }
+
+  const renderHeaderRightElement = () => (
+    <SimpleButton
+      // translator: Used when the QR Code Scanner is opened, and user will manually
+      // enter the information.
+      title={t`Manual info`}
+      onPress={() => navigation.navigate('NanoContractRegisterScreen')}
+    />
+  );
+
+  return (
+    <View style={{
+      flex: 1,
+      justifyContent: 'center',
+      alignItems: 'center',
+      backgroundColor: COLORS.lowContrastDetail,
+      alignSelf: 'stretch',
+    }}
+    >
+      <View style={{
+        flex: 1,
+        justifyContent: 'center',
+        alignItems: 'center',
+        alignSelf: 'stretch',
+      }}
+      >
+        <HathorHeader
+          title={t`Register Nano Contract`.toUpperCase()}
```

Comment:
Can you change this to `Nano Contract Registration`? Or change the one in the previous screen to `Register Nano Contract` so they're consistent when navigating?